third_party/libuv.BUILD

```diff
@@ -324,3 +324,37 @@ cc_library(
         "//visibility:public",
     ],
 )
+
+cc_library(
+    name = "libuv_test",
+    srcs = [
+       "test/test-timer.c",
+       "test/test-timer-again.c",
+       "test/test-timer-from-check.c",
+    ],
+    hdrs = [
+        "test/runner.h",
+        "test/runner-unix.h",
+        "test/task.h",
+    ],
+    includes = [
+        "include",
+        "src",
+    ],
+    deps = [
+        ":libuv"
+    ],
+    linkopts = select({
```

Comment:
The same `copts` would be needed here as well, I think

Reply:
comment addressed